pkg/template_validator.go

```diff
@@ -77,3 +82,10 @@ func isValidImageName(name string) error {
 	}
 	return nil
 }
+
+func hasValidTimeout(timeout int64, name string) error {
```

Comment:
this is an abuse of errors, should really be a bool. But its matching other similar functions so I'm torn :D. *If* you were to change the other `has.* error {` functions to return bool instead would be great :D. If not please open up an issue for it.

Reply:
PR submitted #284 
I think we need to be careful about such convention.